### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by ihorromaniuk**

### CODE REVIEW

File: src/main/java/core/basesyntax/HelloWorld.java [hunk 1]
```diff
@@ -4,4 +4,6 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] arg) {
+    }
 }
```
Comment (liliia-ponomarenko) [inline src/main/java/core/basesyntax/HelloWorld.java:9]: Let’s remove this class and add a regular Main class with testing logic.

File: src/test/java/core/basesyntax/dao/impl/FruitRepositoryImplTest.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        StorageException exception = assertThrows(StorageException.class, () -> {
+            fruitRepository.remove(BANANA, 20);
+        });
+        String actualMessage = exception.getMessage();
+        String expectedMessage = "Not enough banana in storage to remove";
+        assertEquals(expectedMessage, actualMessage);
+    }
+
+    @Test
+    void hasFruit_true_ok() {
```
Comment (liliia-ponomarenko) [inline src/test/java/core/basesyntax/dao/impl/FruitRepositoryImplTest.java:57]: What are you testing here? Let’s  make the method name more informative

File: src/test/java/core/basesyntax/dao/impl/FruitRepositoryImplTest.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        db.put(BANANA, 14);
+        assertTrue(fruitRepository.hasFruit(BANANA));
+    }
+
+    @Test
+    void hasFruit_false_ok() {
```
Comment (liliia-ponomarenko) [inline src/test/java/core/basesyntax/dao/impl/FruitRepositoryImplTest.java:63]: the same

File: src/test/java/core/basesyntax/dao/impl/FruitRepositoryImplTest.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        assertFalse(fruitRepository.hasFruit(BANANA));
+    }
+
+    @Test
+    void getAll_ok() {
```
Comment (liliia-ponomarenko) [inline src/test/java/core/basesyntax/dao/impl/FruitRepositoryImplTest.java:68]: use the same name format

File: src/test/java/core/basesyntax/service/impl/FileServiceImplTest.java [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    private static FileService fileService;
+
+    @BeforeAll
+    static void beforeAll() {
+        fileService = new FileServiceImpl();
+    }
+
+    @Test
+    void read_readFromFile_ok() {
+        List<String> actual = fileService.read("src/test/resources/input.csv");
```
Comment (liliia-ponomarenko) [inline src/test/java/core/basesyntax/service/impl/FileServiceImplTest.java:23]: better make the path a constant variable

File: src/test/java/core/basesyntax/service/impl/ReportGeneratorImplTest.java [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+import core.basesyntax.dao.FruitRepository;
+import core.basesyntax.dao.impl.FruitRepositoryImpl;
+import core.basesyntax.db.Database;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReportGeneratorImplTest {
```
Comment (liliia-ponomarenko) [inline src/test/java/core/basesyntax/service/impl/ReportGeneratorImplTest.java:14]: Let’s  add a test case with empty storage

**Review by liliia-ponomarenko — CHANGES_REQUESTED** [review]
Good job! Let’s fix a few comments ;)

File: src/main/java/core/basesyntax/HelloWorld.java (hunk 1 above)
Comment (IhorRome) [reply src/main/java/core/basesyntax/HelloWorld.java:9]: can't add code here - getting <80% coverage doing so

File: src/main/java/core/basesyntax/Main.java [hunk 7]
```diff
@@ -3,5 +3,8 @@
 /**
  * Feel free to remove this class and create your own.
  */
-public class HelloWorld {
+public class Main {
+    public static void main(String[] arg) {
+
```
Comment (Elena-Bruyako) [inline src/main/java/core/basesyntax/Main.java:8]: there is should be implementation of main method from previous task

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
one comment

**Review by liliia-ponomarenko — APPROVED** [review]
good job! 👍
